conda/resolve.py

```diff
@@ -386,22 +386,6 @@ def ms_depends(self, dist):
             self.ms_depends_[dist] = deps
         return deps
 
-    def depends_on(self, spec, target):
```

Comment:
We really don't use this? I could have sworn we have some sort of `conda search` command that can find packages that depend on another.

Reply:
I couldn't find anywhere that we do.  Not any more anyway.  `conda search` probably doesn't use resolve at all any longer, so maybe that's what happened.